runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/BpmnModeler/BpmnModeler.tsx

```diff
@@ -222,9 +232,17 @@ const BpmnModeler = React.forwardRef<ModelerImperativeHandle, ModelerProps>(
                             />
                             <ImportExportPanel
                                 onExport={onExport}
-                                onImport={(e) => {
-                                    onImport(e);
-                                    setImported(true);
+                                onImport={(e, additionalInfo) => {
+                                    onImport(e, additionalInfo);
+                                    setTimeout(() => {
+                                        const { _elements } =
```

Comment:
why timeout?

Reply:
we need to make sure that modeler is refreshed after import, 200 ms is a sweet spot